docs/man/dune.1.md

```diff
@@ -0,0 +1,291 @@
+---
+title: DUNE
+section: 1
+header: User Manual
+footer: DUNE 0.0.1
+date: April 04, 2023
+---
+# NAME
```

Comment:
also each doc has one title, that starts with one hash # char and then the sub-titles follow and start with two or more hash chars, e.g. ## sub-title 1, #### sub-title N

if `unix man` integration can not be done respecting our current docs general guidelines I guess it is alright for this doc to be merged like this.

Reply:
Yes terminals don't supports different sizes of fonts so it is the only way to write a name of section which will be correctly converted to a format which man utility understand.

Reply:
> in general an ENF/EOS markdown doc has one title,

@iamveritas This is the source file for generating a man page and is not intended for "direct" consumption. It's probably more appropriate to look at the result (`dune.1`) using a man viewer.